initramfs-init.in

```diff
@@ -345,25 +349,29 @@ mount -t devpts -o gid=5,mode=0620,noexec,nosuid devpts /dev/pts
 [ -d /dev/shm ] || mkdir /dev/shm
 mount -t tmpfs -o nodev,nosuid,noexec shm /dev/shm
 
-if [ -n "$dasd" ]; then
+if [ -n "$KOPT_dasd" ]; then
 	for mod in dasd_mod dasd_eckd_mod dasd_fba_mod; do
 		modprobe $mod
 	done
-	for _dasd in $(echo "$dasd" | tr ',' ' ' ); do
+	for _dasd in $(echo "$KOPT_dasd" | tr ',' ' ' ); do
 		echo 1 > /sys/bus/ccw/devices/"${_dasd%%:*}"/online
 	done
 fi
 
-if [ "${s390x_net%%,*}" = "qeth_l2" ]; then
+if [ "${KOPT_s390x_net%%,*}" = "qeth_l2" ]; then
 	for mod in qeth qeth_l2 qeth_l3; do
 		modprobe $mod
 	done
-	_channel="${s390x_net#*,}"
+	_channel="${KOPT_s390x_net#*,}"
 	echo "$_channel" > /sys/bus/ccwgroup/drivers/qeth/group
 	echo 1 > /sys/bus/ccwgroup/drivers/qeth/"${_channel%%,*}"/layer2
 	echo 1 > /sys/bus/ccwgroup/drivers/qeth/"${_channel%%,*}"/online
 fi
 
+if [ -n "$KOPT_ssh_key" ] || [ -n "$KOPT_ssh_pass" ]; then
+	: ${ALPINE_REPO:=http://dl-cdn.alpinelinux.org/alpine/latest-stable/main}
+fi
+
 # load available drivers to get access to modloop media
```

Comment:
I don't like alpine_repo here. IF we set a global default we should do it more general and don't start to repeat it for every network option. I also wonder if we really need to use dl-cdn as the default repo. I think having a more generic name would be more suitable ie main.a.o or dl-default (or any good alternative).